app/models/user.rb

```diff
@@ -37,6 +37,10 @@ def claims
     Claim.fetch_all(vaafi_headers)
   end
 
+  def request_claim_decision(claim_id)
+    Claim.request_decision(vaafi_headers, claim_id)
+  end
```

Comment:
I refactored this a bit in another branch that I will open a PR for once this gets merged in to remove the intermediary calls on the user model. I think eventually I'd like to refactor this again to match Rx a bit more (controller instantiates a client with user auth info) but they are moving over their gem into this repo and potentially refactoring along the way so I'm OK keeping this fairly simple and waiting to make any more changes until we decide if there is a single pattern we want to follow